---
title: "Research Projects"
date: 2020-01-22
categories:
- research
tags:
- portfolio
comments: false
showSocial: false

---

<!--more-->

# Polyploidy and plant genome evolution
I am studying the molecular evolution of post-polyploidization *Brassica* species such as *B. rapa*, *B. olecarea* and *B. napus*.

**Hao Y**, Washburn JD, Rosenthal J, Nielsen B, Lyons E, Edger PP, Pires JC and Conant GC. 2018. [Patterns of population variation in two paleopolyploid eudicot lineages suggest that dosage-based selection on homeologs is long-lived](https://academic.oup.com/gbe/article/10/3/999/4943970). *Genome Biol Evol* **10**(3):999-1011<br>


# Mammalian genome evolution
Hao Y, Lee HJ, Baraboo M, Burch K, Maurer T, Somarelli JA and Conant GC. 2020. [Baby Genomics: tracing the evolutionary changes that gave rise to placentation](https://academic.oup.com/gbe/advance-article/doi/10.1093/gbe/evaa026/5735468). *Genome Biol Evol* (in press)<br>


# Collaborative projects:

#### Development of the [POInT](https://github.com/gconant0/POInT) tool
Emery M, Willis MS, **Hao Y**, Barry K, Oakgrove K, Peng Y, Schmutz J, Lyons E, Pires JC, Edger PP and Conant GC. 2018. [Preferential retention of genes from one parental genome after polyploidy illustrates the nature and scope of the genomic conflicts induced by hybridization](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007267). *PLoS Genet* **14**(3):e1007267<br>

#### Metabolic network analysis for *B. napus* differentially expressed genes
An H, Qi X, Gaynor ML, **Hao Y**, Gebken SC, Mabry ME, McAlvay AC, Teakle GR, Conant GC, Barker MS, Fu T, Yi B and Pires JC. 2019. [Transcriptome and organellar genome sequencing elucidates the origin and diversification of allotetraploid *Brassica napus*](https://www.nature.com/articles/s41467-019-10757-1). *Nat Commun* **10**(1):2878<br>


#### Nematode triploidy
Schoonmaker A, **Hao Y**, Bird DM and Conant GC. 2020. [A single, shared triploidy in three species of parasitic nematodes](https://www.g3journal.org/content/10/1/225.abstract). *G3: Genes, Genomes, Genetics* **10**(1):  225-233

#### SMN phylogeny
Osman EY, Bolding MR, Villal√≥n E, Kaifer KA, Lorson ZC, Tisdale S, **Hao Y**, Conant GC, Pires JC, Pellizzoni L, and Lorson CL. 2019. [Functional characterization of SMN evolution in mouse models of SMA](https://www.nature.com/articles/s41598-019-45822-8). *Sci Rep* **9**(1):9472

#### Ovine ruminal metagenomics, class project for AnSci8633 Molecular Evolution
Wolff SM, Ellison MJ, **Hao Y**, Cockrum RR, Austin KJ, Baraboo M, Burch K, Lee HJ, Maurer T, Patil R, Ravelo A, Taxis TM, Truong H, Lamberson WR, Cammack KM and Conant GC. 2017. [Diet shifts provoke complex and variable changes in the metabolic networks of the ruminal microbiome](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-017-0274-6). *Microbiome* **5**(1):60

#### Consulting project for ST810 Bioinformatics Consulting
Differential gene expression in the brains of wild island and laboratory mouse strains. In consultation with Nicolette Petervary and Dr. John Godwin.<br>

# Hackathon projects
#### [clustifyR](https://rnabioco.github.io/clustifyR/)
An R package that classifies cells and clusters in single-cell RNA sequencing experiments using reference bulk RNA-seq data sets, gene signatures or marker genes. <br>

Fu R, Gillen AE, Sheridan RM, Tian C, Daya M, **Hao Y**, Hesselberth JR and Riemondy KA. 2020. [clustifyr: An R package for automated single-cell RNA sequencing cluster classification](https://www.biorxiv.org/content/10.1101/855064v1.full). (In review)

#### [MR_BACOn](https://github.com/NCBI-Hackathons/MR_BACOn)
An R Shiny app for Mendelian Randomization with Biomarker Associations for Causality with Outcomes. <br>


